server/src/test/java/com/objectcomputing/checkins/services/fixture/PermissionFixture.java

```diff
@@ -10,83 +10,75 @@
-        Permissions.CAN_CREATE_FEEDBACK_REQUEST,
-        Permissions.CAN_DELETE_FEEDBACK_REQUEST,
-        Permissions.CAN_VIEW_FEEDBACK_ANSWER,
-        Permissions.CAN_DELETE_ORGANIZATION_MEMBERS,
-        Permissions.CAN_CREATE_ORGANIZATION_MEMBERS,
-        Permissions.CAN_VIEW_ROLE_PERMISSIONS,
-        Permissions.CAN_VIEW_PERMISSIONS,
-        Permissions.CAN_VIEW_SKILLS_REPORT,
-        Permissions.CAN_VIEW_RETENTION_REPORT,
-        Permissions.CAN_VIEW_ANNIVERSARY_REPORT,
-        Permissions.CAN_VIEW_BIRTHDAY_REPORT,
-        Permissions.CAN_VIEW_PROFILE_REPORT,
-        Permissions.CAN_CREATE_CHECKINS,
-        Permissions.CAN_VIEW_CHECKINS,
-        Permissions.CAN_UPDATE_CHECKINS,
-        Permissions.CAN_ASSIGN_ROLE_PERMISSIONS,
-        Permissions.CAN_VIEW_SKILL_CATEGORIES,
-        Permissions.CAN_EDIT_SKILL_CATEGORIES
+    List<Permission> adminPermissions = List.of(
+        Permission.CAN_VIEW_FEEDBACK_REQUEST,
+        Permission.CAN_CREATE_FEEDBACK_REQUEST,
+        Permission.CAN_DELETE_FEEDBACK_REQUEST,
+        Permission.CAN_VIEW_FEEDBACK_ANSWER,
+        Permission.CAN_DELETE_ORGANIZATION_MEMBERS,
+        Permission.CAN_CREATE_ORGANIZATION_MEMBERS,
+        Permission.CAN_VIEW_ROLE_PERMISSIONS,
+        Permission.CAN_VIEW_PERMISSIONS,
+        Permission.CAN_VIEW_SKILLS_REPORT,
+        Permission.CAN_VIEW_RETENTION_REPORT,
+        Permission.CAN_VIEW_ANNIVERSARY_REPORT,
+        Permission.CAN_VIEW_BIRTHDAY_REPORT,
+        Permission.CAN_VIEW_PROFILE_REPORT,
+        Permission.CAN_CREATE_CHECKINS,
+        Permission.CAN_VIEW_CHECKINS,
+        Permission.CAN_UPDATE_CHECKINS,
+        Permission.CAN_ASSIGN_ROLE_PERMISSIONS,
+        Permission.CAN_VIEW_SKILL_CATEGORIES,
+        Permission.CAN_EDIT_SKILL_CATEGORIES
     );
 
-    default Permission createACustomPermission(Permissions perm) {
-        return getPermissionRepository().save(new Permission(null, perm.name(), perm.getDescription()));
-    }
 
-    default void saveAllPermissions() {
-        for(Permissions permissions : Permissions.values()) {
-            getPermissionRepository().save(new Permission(null, permissions.name(), permissions.getDescription()));
-        }
-    }
 
     default void setPermissionsForAdmin(UUID roleID) {
-        List<Permission> permissions = getPermissionRepository().findAll();
-        for(Permissions adminPermission : adminPermissions) {
-            Optional<Permission> permission = permissions.stream().filter(s -> s.getPermission().equals(adminPermission.name())).findFirst();
-            permission.ifPresent(value -> setRolePermission(roleID, value.getId()));
+        List<Permission> permissions = List.of(Permission.values());
+        for(Permission adminPermission : adminPermissions) {
+            Optional<Permission> permission = permissions.stream().filter(s -> s.name().equals(adminPermission.name())).findFirst();
+            permission.ifPresent(value -> setRolePermission(roleID, value));
```

Comment:
Iterating over all the permissions no longer seems necessary now that the implementation is using only an enum. The check on line 64 will always return a permission since the list of `adminPermissions` is guaranteed to only contain a subset of the enum. (This applies to `setPermissionsForPdl` and `setPermissionsForMember` as well)